codemods/next/13/ab-test-boilerplate/README.md

```diff
@@ -0,0 +1,31 @@
+# Generate URL Patterns for Cal.com
+
+## Description
+
+This codemod generates a boilerplate code for ab tests using next js middlewares. Middleware controls the redirects from legacy to future page. redirect probability and targeted pages can be controlled by env variables. 
```

Comment:
boilerplace code (without a)
A/B tests
Next.js
The middleware controls
the legacy to the future pages.
You can control the redirection probability and targeted pages using environment variables.